### ISSUE TITLE
Personal Desk Booking Page - UI

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#12]
## User story
1. As a user
2. I want / need to see a UI page for booking personal desks
3. So that I can select a date and time as well as select my desk preferences

## Acceptance criteria
* The UI page for booking personal desks should be identifyable as such (e.g header with: Hello username, book a personal desk)
* The page should provide a calendar to select a date and an option to enter a time for booking desks
* The page should offer options to select preferences, e.g country, location, building, room, desk properties
* The page should contain a search option to initialize the booking search process
* The page should be extensible regarding later features

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
